docs/hybrid.md

```
@@ -0,0 +1,309 @@
+**Use Container Mode:**
+- ✅ Legacy compatibility
+- ✅ Using override provider settings
+- ✅ Strict security policies requiring full containerization
+
+## Quick Start
+
+### Basic Usage
+
+```bash
+# Hybrid mode is default - supports both source and binary analysis
+kantra analyze --input /path/to/app --output ./output
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h5

(MD001, heading-increment)